### ISSUE TITLE
Appendix D (Accessibility): needs text

### ISSUE THREAD

**Issue by davidcarlisle** [w3c/mathml#346]

**Comment by Steve-Noble** [comment]
If we wanted to follow the example of the [EPUB 3 Spec](https://www.w3.org/TR/epub-33/), we could add a brief section called "Accessibility" (in the EPUB spec, it is Section 10), and then point to a separate MathML Accessibility specification which lives independent from the full MathML spec. In that case we would not need to have Accessibility as an appendix.

I provide some text for such a brief section below. However, if there is opposition to this approach, then the text below can be repurposed for the intro of the accessibility appendix, if we must really go to that approach. Nonetheless, having "MathML Accessibility" as its own independent specification may have several benefits.

Here's the text...

### Accessibility

As an essential element of the Open Web Platform, the W3C MathML specification has the unprecedented potential to enable content authors and developers to incorporate mathematical expressions on the web in such a way that the underlying structural and semantic information can be exposed to other technologies. Enabling this information exposure is foundational for accessibility, as well as providing a path for making digital mathematics content machine readable, searchable and reusable.

The internationally accepted standards and underpinning principles for creating accessible digital content on the web can be found in the W3C's Web Content Accessibility Guidelines (WCAG) [[wcag2](https://www.w3.org/TR/WCAG2/)]. In extending these principles to digital content containing mathematical information, WCAG provides a useful framework for defining accessibility wherever MathML is used.

As the current WCAG guidelines provide no direct guidance on how to ensure mathematical content encoded as MathML will be accessible to users with disabilities, a separate specification, MathML Accessibility [mathml-a11y], defines how to apply these guidelines to digital content containing MathML.

This specification recommends that digital content containing MathML conform to the accessibility requirements defined in [mathml-a11y]. A benefit of following this recommendation is that it helps to ensure that digital mathematics content meets the accessibility requirements already widely used around the world for web content. In addition, ensuring that digital mathematics materials are accessible will expand the readership of such content to both readers with and without disabilities.

#### NOTE
This specification does not integrate the accessibility requirements to allow them to adapt and evolve independent of the MathML specification — accessibility practices often need more frequent updating. The accessibility specification is also intended for use with past, present, and future versions of MathML, in addition to considerations for both the MathML-Core and the full MathML specification. The approach of a separate specification ensures that the evolution of MathML does not lock accessibility in time, and allows content authors to apply the most recent accessibility requirements.

**Comment by NSoiffer** [comment]
While there are definite advantages to separating out accessibility, we
can't push off/point to the requirements to a document that doesn't exist.
Having a second document means more overhead and thus more time spent,
something that seems in very short supply. I think that in balance,
including accessibility statements in an appendix is a better path forward
at this point in time.

    Neil

**Comment by davidcarlisle** [comment]
I'd agree with @NSoiffer here. We are already getting critically short of time in the current charter, and still have to get Accessibility (and other) text for Full, Core and the Notes. I don't think we have the time or resources to do a fourth document.  It also isn't clear if there will be enough text to justify a separate document. Parts of the epub document relate to implementation requirements of the ebook User Interface which don't apply to MathML which sits in a host document and relies on the host  user interface.
  
That said, if the appendix gets too big, as we have just seen with media types, it is not too hard to extract an appendix to a separate document, so that decision can be deferred until later.

**Comment by davidcarlisle** [comment]
with respect to the proposed comment about the separate doc applying to Full and Core, hopefully the Notes can fill some of that role. They are non-normative but many guidelines and best practices may be suitable for the Notes, and more easily updated than a rec-track document (whether that is mathml or a separate accessibilty rec)

**Comment by Steve-Noble** [comment]
In that case, I can revise what I wrote as the intro to the Accessibility appendix, and keep adding text. In regards to a separate document on a Note track, I could work with APA to craft something like "Mathematics Accessibility Users Requirements." APA has created a whole slew of such Notes, as you can see at https://github.com/w3c/apa/blob/main/README.md

That would allow the MathML spec to proceed on its current course and timeline without adding additional time.

**Comment by davidcarlisle** [comment]
I think we need to prioritize the existing Full, Notes and Core documents. If the accessibility chapter in Notes on MathML gets to be large, only then should we consider if we need a fourth document specifically on MathML accessibility. It should be easy at that point to move sections of the notes to a new document, so work is not lost.

Curently the main concern is the accessibility appendix in Full which is literally empty, without any words,  just a couple of issue links.

**Comment by Steve-Noble** [comment]
Forgive my inability to fully understand your points, but should the guidance on MathML accessibility go into the Notes document you are referring to, or should it go in the Appendix, or should the Appendix on Accessibility be only a short section which points the reader to a fuller discussion in the Notes section?

For instance, I believe most of Neil's previous guidance within the [MathML in DAISY Structure Guidelines](https://www.daisy.org/z3986/structure/SG-DAISY3/part2-math.html) are still valid in concept to HTML documents containing MathML. Obviously, we cannot use them as is because they are explicit for DAISY XML documents, but a good bit of that guidance can be repurposed and revised to be explicit to HTML+MathML. So, if I use that model, would all this go in the Accessibility Appendix or the Notes section?

**Comment by davidcarlisle** [comment]
@Steve-Noble   as a general rule, for the whole document not just accessibility, we are aiming to have much less non-normative discursive text in Mathml4 than we had in Mathml3, and move that non normative text to the notes document (which has not, so far not had a lot of attention but does hold some collected texts moved from Mathml3 or repurposed from mathml-refresh community group documents.)

So anything normative or almost normative "best practice" should probably be in Mathml4, and general background and motivation material should be in the notes, but there is no clear dividing line.

But at this point it _really doesn't matter_  You could put any text into Mathml4.  It is easy to move text between documents in later edits. What is concerning now is that we are 14 months into a 2 year charter and we have not yet added a single word to any document accessibility section.

**Comment by Steve-Noble** [comment]
@davidcarlisle In that case, I'll just dump everything into the Appendix for now, and others can decide what needs to go where. As long as it just isn't discarded off hand, that should be fine. I can proceed with adapting the previously mentioned DAISY Structure Guidelines, and fold in anything else that seems appropriate by extending WCAG 2.1 to mathematics content. 

If that is going to be seen as the wrong approach, then I need to know now before investing any more of my time.

**Comment by davidcarlisle** [comment]
@Steve-Noble  that sounds like a good plan, thanks. Once there is text that can be reviewed, it's easy to get an opinion if some sections are too wordy and could be split to another document, but while the document is literally empty it's hard to have an opionion on it at all, other than its emptiness:-)

**Comment by davidcarlisle** [comment]
basic appendix structure now in place and individual issues tracked with [accessibility](https://github.com/w3c/mathml/labels/accessibility) label, so closing here
